.github/pull_request_template.md

```diff
@@ -3,16 +3,17 @@
 Submitting a Pull Request
 
 - Choose a meaningful title for this pull request.
-- Please fill out all sections of this form. You can delete the helpful comments.
+- Please fill out all sections of this form. You can delete the placeholder comments.
 - Pull Requests without clear information will take longer and may even be rejected.
+- **Note:** Pull requests should be created from **[confirmed]** issues tracked in the issue management board.
```

Comment:
I mean they can create a PR to any issue, it's just their risk that it might not be looked at.

Reply:
yes, I mentioned that in the contributing template, no need to duplicate imo.